src/components/LoanCards/GridLoanCard.vue

```diff
@@ -164,6 +164,13 @@ export default {
 		},
 		trackInteraction(args) {
 			this.$emit('track-interaction', args);
+			if (args?.interactionType === 'addToBasket' && this.getTagInfo()) {
+				this.$kvTrackEvent(
+					'loan-card',
+					'add-to-basket',
+					this.getTagInfo()
```

Comment:
We analytics labels should always be kebab-case -- so these should be 'ending-soon' etc